app/Repositories/Admin/Project/ProjectRepository.php

```diff
@@ -234,4 +239,8 @@ public function updateUserStarterFlag(): string
         $this->model->whereIn('cloned_from', $starterProjects->modelKeys())->update(['is_user_starter' => true]);
         return 'Existing records are updated successfully for user starter project flag.';
     }
+
+    public function getNonStarterProjects(){
+
+    }
```

Comment:
What does this?

Reply:
Ooh! I forgot to remove this. Instead of using this function I simple put the extra condition in get All function. I'll remove this.